### PR TITLE
removing references to deprecated killQuery and killQueries procedures

### PR DESCRIPTION
**Pull request by phil198**
Removed references to

- dbms.killQuery(<query-id>)
- dbms.killQueries(<query-ids>)

### CODE REVIEW

**Comment by phil198** [comment]
[Remove deprecated procedures for listing and terminating transactions and queries](https://trello.com/c/fofpFudX/2749-remove-deprecated-procedures-for-listing-and-terminating-transactions-and-queries)

File: cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc [hunk 1]
```diff
@@ -1599,7 +1599,7 @@ label:updated[]
 queryId
 ----
 a|
-The `queryId` procedure format has changed, and no longer includes the database name. For example, `mydb-query-123` is now `query-123`. This change affects built-in procedures `dbms.listQueries()`, `dbms.listActiveLocks(queryId)`, `dbms.killQueries(queryIds)` `and dbms.killQuery(queryId)`.
```
Comment (mnd999) [inline cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc]: This is from a list of things that changed in 4.1 - I'm not sure if we actually need to change this. Would be interesting to see what the docs team think.

**Comment by renetapopova** [comment]
Thanks, @phil198. Unfortunately, I can't build it locally and the build is red in TC. So, we have to wait until the checks are able to pass successfully.

File: cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc (hunk 1 above)
Comment (renetapopova) [reply cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc]: I don't think we should change it here as it is still valid for the 4.1 version, right?

Comment (phil198) [reply cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc]: thats correct, yes.

Comment (renetapopova) [reply cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc]: We should add them (`dbms.listQueries` and `dbms.killQueries()`) to the first section **Version 5.0 ->  Removed features** if not already added.

Comment (phil198) [reply cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc]: makes sense, thanks

**Comment by phil198** [comment]
> Thanks, @phil198. Unfortunately, I can't build it locally and the build is red in TC. So, we have to wait until the checks are able to pass successfully.

all green now 😄

**Comment by phil198** [comment]
ok to merge now @renetapopova ? Thanks

**Comment by renetapopova** [comment]
Thanks, @phil198. Still having problems building it locally to see how it looks. I will merge it when I solve this issue.

**Comment by phil198** [comment]
Thanks @renetapopova

**Review by renetapopova — APPROVED** [review]
LGTM! Thanks.

**Comment by phil198** [comment]
cheers :)

**Pull request merged**